.github/workflows/triage.yml

```diff
@@ -219,12 +219,92 @@ jobs:
               content: system "swift", "build"
 
             - label: long build
-              path: Formula/.+/(agda|apache-pulsar|arangodb|aws-sdk-cpp|cp2k|deno|dotnet|emscripten|envoy|gcc|ghc|graph-tool|gstreamer|haskell-language-server|libtensorflow|mame|metashell|mlkit|mpich|node|nwchem|ponyc|pytorch|rust|swift|texlive|qt|root|souffle|v8|vtk|)(@[0-9]+)?.rb
+              path: >
```

Comment:
Using `>` will replace the newlines with spaces, so this doesn't work, unfortunately 😞

Reply:
:( but `path: |` would work?

Reply:
Nope, `|` retains new lines: https://yaml-multiline.info

Reply:
We could probably just strip/collapse new lines in the action. Real file names shouldn't really have them.

Reply:
Reading trough the yaml documentation, I think we want `>-` which strips newlines and doesn't add one at the end.

Reply:
`>-` just strips the final `\n`, but intermediate ones are still replaced with spaces.

Reply:
Ok so I think the only option left is to keep the double quote string and double escape characters within the string.